prisma/schema.prisma

```diff
@@ -91,3 +92,17 @@ model TrainingRequest {
 
   @@map("training_requests")
 }
+
+model Notification {
+  id        String   @id @default(auto()) @map("_id") @db.ObjectId
+  userId    String   @db.ObjectId
+  user      User     @relation(fields: [userId], references: [id])
+  avatar    String?
```

Comment:
avatar is the user's avatar right? .You can get the user's avatar from user's collections